src/Components/test/E2ETest/Tests/InteropTest.cs

```diff
@@ -124,7 +133,7 @@ public void CanInvokeDotNetMethods()
             var interopButton = Browser.FindElement(By.Id("btn-interop"));
             interopButton.Click();
 
-            var wait = new WebDriverWait(Browser, TimeSpan.FromSeconds(10))
+            var wait = new WebDriverWait(Browser, TimeSpan.FromSeconds(1000))
```

Comment:
Is there a functional reason we need to wait this long or was this done to accomodate some timeouts?

Reply:
Nope, that was for debugging reasons, I'll revert it. Thanks!